internal/app/tfsec/cmd/output.go

```diff
@@ -43,15 +43,23 @@ func gatherLinks(result rules.Result) []string {
 	if result.Rule().Terraform != nil {
 		links = result.Rule().Terraform.Links
 	}
-	return append([]string{
-		fmt.Sprintf(
-			"https://aquasecurity.github.io/tfsec/%s/checks/%s/%s/%s/",
-			v,
-			result.Rule().Provider,
-			strings.ToLower(result.Rule().Service),
-			result.Rule().ShortCode,
-		),
-	}, links...)
+
+	var docsLink []string
+	if result.Rule().Provider == "custom" {
```

Comment:
```suggestion
	if result.Rule().Provider == provider.CustomProvider {
```

Nitpick but can we use the enum

Reply:
ah yes, thanks :)